crds/workspace.devfile.io_devworkspaces.v1beta1.yaml

```diff
@@ -31,3936 +18,8086 @@ spec:
+                                        items:
+                                          properties:
+                                            attributes:
+                                              additionalProperties:
+                                                type: string
+                                              description: "Map of implementation-dependant
+                                                string-based free-form attributes.
+                                                \n Examples of Che-specific attributes:
+                                                \n - cookiesAuthEnabled: \"true\"
+                                                / \"false\", \n - type: \"terminal\"
+                                                / \"ide\" / \"ide-dev\","
+                                              type: object
+                                            exposure:
+                                              description: "Describes how the endpoint
+                                                should be exposed on the network.
+                                                \n - `public` means that the endpoint
+                                                will be exposed on the public network,
+                                                typically through a K8S ingress or
+                                                an OpenShift route. \n - `internal`
+                                                means that the endpoint will be exposed
+                                                internally outside of the main workspace
+                                                POD, typically by K8S services, to
+                                                be consumed by other elements running
+                                                on the same cloud internal network.
+                                                \n - `none` means that the endpoint
+                                                will not be exposed and will only
+                                                be accessible inside the main workspace
+                                                POD, on a local address. \n Default
+                                                value is `public`"
+                                              enum:
+                                              - public
+                                              - internal
+                                              - none
+                                              type: string
+                                            name:
+                                              type: string
+                                            path:
+                                              description: Path of the endpoint URL
+                                              type: string
+                                            protocol:
+                                              description: "Descr
```

Comment:
those fields added in v1alpha2 should not be included in v1alpha1? similar as other files.